constraint/pkg/client/client.go

```diff
@@ -622,7 +622,7 @@ func (c *Client) RemoveData(ctx context.Context, data interface{}) (*types.Respo
 // Review makes sure the provided object satisfies all stored constraints.
 // On error, the responses return value will still be populated so that
 // partial results can be analyzed.
-func (c *Client) Review(ctx context.Context, obj interface{}, opts ...drivers.QueryOpt) (*types.Responses, error) {
+func (c *Client) Review(ctx context.Context, obj interface{}, sourceEP string, opts ...drivers.QueryOpt) (*types.Responses, error) {
```

Comment:
@maxsmythe do we want to move `QueryOpt` under client then? How do you imagine passing `QueryOpt` to `review`?

Reply:
Yeah, client probably needs to own QueryOpt, or it should be in a separate package, since putting it in client may lead to an import cycle.

> How do you imagine passing QueryOpt to review?

I'm a bit confused by this question. Isn't it already being passed to Review()?

Reply:
Yip! but as a part of refactor we will not be passing the same to `Review`. I found a way to set those options and pass it to `review`.